purchase_request_to_rfq/models/purchase_request.py

```diff
@@ -60,8 +61,10 @@ def _compute_purchase_state(self):
                     temp_purchase_state = 'draft'
             rec.purchase_state = temp_purchase_state
 
-    purchased_qty = fields.Float(string='Quantity in RFQ or PO',
-                                 compute="_compute_purchased_qty")
+    purchased_qty = fields.Float(
+        string='Quantity in RFQ or PO',
+        digits=dp.get_precision('Product Unit of Measure'),
```

Comment:
Are you sure this one is needed? as it is inherited if should already have digits defined.

Reply:
Nevermind, they are different fields... my bad.